SmilesMiles/Model/TripModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  TripModel.swift
+//  SmilesMiles
+//
+//  Created by Pavan Sai Nallagoni on 5/27/23.
+//
+
+struct Trip: Codable, Equatable {
+    let tripID: String
+    let tripName: String
+    let status: String
```

Comment:
Check if status needs to be var, as status might change from In progress to Complete